packages/target-nodejs-sdk/src/api/index.js

```diff
@@ -0,0 +1,88 @@
+import {
+  DECISIONING_METHOD,
+  requiresDecisioningEngine
+} from "@adobe/target-tools";
+import { createOnDeviceApi } from "./ondevice";
+import { Configuration as AepEdgeConfiguration } from "../../../../../aep-edge-nodejs-sdk/packages/aep-edge-tools/aep-edge-api-client";
+import { isBeaconSupported, isNonEmptyString } from "../utils";
+import {
+  createBeaconDeliveryApi,
+  createDeliveryApi,
+  createDeliveryConfiguration
+} from "./delivery";
+import { createAepApi, createAepEdgeConfiguration } from "./aep";
+
+function createRemoteApi(sdkConfig, configuration, useBeacon) {
+  if (configuration instanceof AepEdgeConfiguration) {
+    return createAepApi(sdkConfig, configuration);
+  }
+
+  return useBeacon && isBeaconSupported()
```

Comment:
We support it because the node.js sdk also produces a browser build that some people use instead of at.js.  In that case it leverages beacon for notifications.

Reply:
Ok, I think we should implement a way to detect when the browser build is used and change the SDK name header to `AdobeTargetNodeBrowser` in this case, in a separate task.